projects/storefrontlib/src/lib/store-finder/store-finder.module.ts

```diff
@@ -2,15 +2,9 @@ import { NgModule } from '@angular/core';
 import { ReactiveFormsModule } from '@angular/forms';
 import { CommonModule } from '@angular/common';
 import { RouterModule } from '@angular/router';
-import { StoreModule } from '@ngrx/store';
-import { EffectsModule } from '@ngrx/effects';
 
 import { CmsModule } from '../cms/cms.module';
 
-import { effects } from './store/effects/index';
-import { reducers } from './store/reducers/index';
-import { services } from './services/index';
-
 import { StoreFinderSearchComponent } from './components/store-finder-search/store-finder-search.component';
```

Comment:
Can we reorganize all components? Now, all components are in flat structure.

Reply:
Maybe let's do it as a scope of some refactoring task.